### PR TITLE
docs(router): fix documentation examples

### PR DESCRIPTION
**Pull request by mexx**

### CODE REVIEW

File: modules/@angular/router/src/interfaces.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ export interface CanActivate {
  *
  * ```
  * @Injectable()
- * class CanActivateTeam implements CanActivate {
+ * class CanActivateTeam implements CanActivateChild {
```
Comment (vicb) [inline modules/@angular/router/src/interfaces.ts]: update l 74  ?

Comment (mexx) [inline modules/@angular/router/src/interfaces.ts]: Sorry, but what exactly are you pointing at?

Comment (sirajc) [inline modules/@angular/router/src/interfaces.ts]: Update line 74

Comment (mexx) [inline modules/@angular/router/src/interfaces.ts]: Sure, the only thing I don't get, how should it be changed, I can't see anything problematic in it.

**Comment by alxhub** [comment]
Hi @Mexx, sorry for the slow review process. Can you rebase this please? Thanks.

**Comment by vsavkin** [comment]
Fix is in master

**Pull request closed without merging**